src/valkey-benchmark.c

```diff
@@ -1817,13 +1851,16 @@ int parseOptions(int argc, char **argv) {
         " --cert <file>      Client certificate to authenticate with.\n"
         " --key <file>       Private key file to authenticate with.\n"
         " --tls-ciphers <list> Sets the list of preferred ciphers (TLSv1.2 and below)\n"
-        "                    in order of preference from highest to lowest separated by colon (\":\").\n"
-        "                    See the ciphers(1ssl) manpage for more information about the syntax of this string.\n"
+        "                    in order of preference from highest to lowest\n"
+        "                    separated by colon (\":\"). See the ciphers(1ssl)\n"
+        "                    manpage for more information about the syntax of\n"
+        "                    this string.\n"
```

Comment:
Nice to see you have fixed the width of the complete help output.

(Not that it matters much but) here it looks like the output is narrower than 80. The source code has indendation and quotation marks, so the source code lines can be 91 (indentation 8 + opening `"` + closing `\n"`) if we want the output to be 80. Am I counting correctly like this?

```suggestion
        "                    in order of preference from highest to lowest separated by\n"
        "                    colon (\":\"). See the ciphers(1ssl) manpage for more\n"
        "                    information about the syntax of this string.\n"
```